src/internal/function.go

```
@@ -212,6 +212,17 @@ func removeElementByValue(slice []string, value string) []string {
 	return newSlice
 }
 
+func isValidFileName(name string) bool {
+	switch {
+	case name == ".", name == ".":
```

golangci-lint (1.64.8)

dupCase: 'case name == "."' is duplicated

(gocritic)